python/dask_cudf/dask_cudf/backends.py

```diff
@@ -625,13 +625,68 @@ def read_csv(*args, **kwargs):
 
     @staticmethod
     def read_hdf(*args, **kwargs):
-        from dask_cudf import from_dask_dataframe
-
         # HDF5 reader not yet implemented in cudf
         warnings.warn(
             "read_hdf is not yet implemented in cudf/dask_cudf. "
             "Moving to cudf from pandas. Expect poor performance!"
         )
-        return from_dask_dataframe(
-            _default_backend(dd.read_hdf, *args, **kwargs)
+        return _default_backend(dd.read_hdf, *args, **kwargs).to_backend(
+            "cudf"
+        )
+
+
+# Define "cudf" backend entrypoint for dask-expr
+class CudfDXBackendEntrypoint(DataFrameBackendEntrypoint):
```

Comment:
There is nothing "new" here. We have already defined/registered a `DataFrameBackendEntrypoint` class for legacy dask-dataframe collections. Now we need to define/register a new one class for new expr-based collections. The difinitions will look very similar, but the classes need to be different.